src/cascadia/ShellExtension/OpenTerminalHere.cpp

```diff
@@ -130,12 +130,11 @@ HRESULT OpenTerminalHere::GetIcon(IShellItemArray* /*psiItemArray*/,
 try
 {
     std::filesystem::path modulePath{ wil::GetModuleFileNameW<std::wstring>(wil::GetModuleInstanceHandle()) };
-    modulePath.replace_filename(WindowsTerminalExe);
     // WindowsTerminal.exe,-101 will be the first icon group in WT
     // We're using WindowsTerminal here explicitly, and not wt (from GetWtExePath), because
     // WindowsTerminal is the only one built with the right icons.
-    const auto resource{ modulePath.wstring() + L",-101" };
-    return SHStrDupW(resource.c_str(), ppszIcon);
+    modulePath.replace_filename(L"WindowsTerminal.exe,-101");
+    return SHStrDupW(modulePath.c_str(), ppszIcon);

```

Comment:
This avoids an extra copy just to append the `,-101`.

Reply:
_clever_. I like it.